diremanwolf/CLI-multiple-themes
# Feat: modify-CLI

#### What does this PR do?
This PR includes node packages, changes to project's core and its components, and SASS examples to carry out tests.

#### Where should the reviewer start?
* `src/index.js`
* `src/commands/Generate.js`
* `src/commands/Initialize.js`
* `assets/scripts/Webpack.Config.js`
* `assets/styles/*`

#### How should this be manually tested?
* Clone this branch
* Run `$ npm install`
* Run `$ cli-mutilple-themes start`; this creates a file, `webpack.config.js`, from a template with the configuration it was given.
* Run `$ cli-multiple-themes build`; this creates the CSS used by `index.html`.
* Open `index.html` on your browser; it should have the compiled CSS.
